src/time.jl

```diff
@@ -23,13 +23,9 @@ function every(dt)
     n
 end
 
-function weakrefdo(ref, yes, no=()->nothing)
```

Comment:
I was using this in `fpswhen` below as well.

I added this function back and tried `node = nothing` after `send_value!` in core.jl. The fps node doesn't go away still.